src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -103,7 +105,14 @@ public void check(boolean state) {
      * @see #clickAndWaitToBecomeStale(Duration)
      */
     public void click() {
-        resolve().click();
+        WebElement we = resolve();
+        // button may be obscured by say the "Save Apply" screen so we wait as Selenium will do a scroll but the CSS 
+        // can take a while to update the layout \o/
+        waitFor(we).
+               withTimeout(time.seconds(1), TimeUnit.MILLISECONDS).
```

Comment:
This seems a lot of code to say 1 second.
Maybe `Duration.ofSeconds(1)` if it's the same ? I am not sure on the duration just by reading.

Reply:
`Duration` causes a different `WaitFor`.  obvious isn't it :)

Reply:
this is wrong.  we already use the elastic time in the `waitFor` so do not need to apply it twice.